### ISSUE TITLE
Lokaler Admin

### ISSUE THREAD

**Issue by kallipick** [binary-butterfly/shared-delivery#59]
Testuser https://lokalwirkt.de/admin/user/36/edit

Mit den Rechten kann ich außerhalb der einen Verwaltung Daten der Regionen oder Geschäfte ändern bzw. Löschen.
User hat nur Bad Münstereifel zugeordnet. Kann trotzdem die Region Bad Honnef ändern oder ein Geschäft in Königswinter.

**Comment by the-infinity** [comment]
Ja. Eigentlich sollten Städte nur Moderatorrechte kriegen, keine Adminrechte. Der Zusammenhang mit der Kommune ist nur im Kontext der Moderationsfähigkeit aktuell sinnvoll, Admin bedeutet komplett-Admin.
